docs/dev-docker/README.md

```diff
@@ -261,7 +261,8 @@ Benchmark Meta-Llama-3.1-405B FP8 with input 128 tokens, output 128 tokens and t
     --num-scheduler-steps 10 \
     --tensor-parallel-size 8 \
     --input-len 128 \
-    --output-len 128 
+    --output-len 128 \
+    --enable-chunked-prefill false
```

Comment:
This is now the default on ROCm platform

Reply:
when was the default set?

Reply:
https://github.com/ROCm/vllm/pull/324